dnsop-structured-dns-error-page.txt

```diff
@@ -457,7 +457,24 @@ Internet-Draft            Data for Filtered DNS                June 2022
       create a new EDE option that conveys the information in the "c",
       "s" and "j" fields encoded in the JSON object.
 
-5.  Examples
+5.  Interoperation with RPZ Servers
+
+   This section discusses operation with an RPZ server
+   [I-D.ietf-dnsop-dns-rpz] that indicates filtering with a NXDOMAIN
+   response with the Recursion Available bit cleared (RA=0).
```

Comment:
I see other specifications referring to https://en.wikipedia.org/w/index.php?title=Response_policy_zone&oldid=960043728 instead of the above expired draft. Please see https://www.rfc-editor.org/rfc/rfc8976.html#section-1.4.3.